content/docs/setup/kubernetes/quick-start-gke-dm.md

```diff
@@ -226,23 +226,6 @@ http://localhost:8088/dotviz
 
 For more details, see [About the ServiceGraph Add-on](/docs/tasks/telemetry/servicegraph/#about-the-servicegraph-add-on).
 
-## Tracing
```

Comment:
Tracing should already be included, why remove this? both istio-demo.yaml and istio-demo-auth.yaml should already include tracing.

Reply:
Perhaps I'm missing something in this guide: https://istio.io/docs/setup/kubernetes/quick-start-gke-dm/ 
But following the guide, I'll click this link: https://accounts.google.com/signin/v2/identifier?service=cloudconsole&continue=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/release-0.8/install/gcp/deployment_manager/istio-cluster.jinja&followup=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/master/install/gcp/deployment_manager/istio-cluster.jinja&flowName=GlifWebSignIn&flowEntry=ServiceLogin

When deploying that config Jaeger isn't deployed, and hence the information in the Tracing chapter will not work. It would be good to include the tracing in this example, perhaps an update to the jinja file to include tracing (Jaeger) should be done? 
https://github.com/istio/istio/blob/master/install/gcp/deployment_manager/istio-cluster.jinja